### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by kasya**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -12039,32 +12039,36 @@ msgstr ""
...
+msgstr "<віддалене-призначення>"
 
 #: builtin/pull.c:466 builtin/pull.c:481 builtin/pull.c:486 scalar.c:413
 msgid "<branch>"
 msgstr "<гілка>"
 
 #: builtin/pull.c:474 builtin/rebase.c:1028
 msgid "There is no tracking information for the current branch."
-msgstr ""
+msgstr "Для поточної гілки немає інформації для відстеження."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Немає інформації для відстежування поточної гілки."
```

File: po/uk.po [hunk 2]
```diff
@@ -12039,32 +12039,36 @@ msgstr ""
...
 
 #: builtin/pull.c:483
 msgid ""
 "If you wish to set tracking information for this branch you can do so with:"
 msgstr ""
+"Якщо ви бажаєте встановити інформацію про відстеження для цієї гілки, ви "
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Якщо ви бажаєте встановити інформацію для відстежування цієї гілки, ви "
```

File: po/uk.po [hunk 3]
```diff
@@ -12538,62 +12567,67 @@ msgid ""
...
+"git read-tree [(-m [--trivial] [--aggressive] | --reset | --"
+"prefix=<префікс>)\n"
+"              [-u | -i]] [--index-output=<файл>] [--no-sparse-checkout]\n"
+"              (--empty | <деревоподібне-джерело1> [<деревоподібне-джерело2> "
+"[<деревоподібне-джерело3>]])"
 
 #: builtin/read-tree.c:124
 msgid "write resulting index to <file>"
-msgstr ""
+msgstr "записати отриманий індекс в <файл>"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "записати отриманий індекс у <файл>"
```

File: po/uk.po [hunk 4]
```diff
@@ -12538,62 +12567,67 @@ msgid ""
...
+msgstr "Злиття"
 
 #: builtin/read-tree.c:131
 msgid "perform a merge in addition to a read"
 msgstr "виконати злиття на додачу до читання"
 
 #: builtin/read-tree.c:133
 msgid "3-way merge if no file level merging required"
-msgstr ""
+msgstr "3-стороннє злиття, якщо не потрібне злиття на рівні файлів"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "3-стороннє злиття, при відсутності потреби в злитті на рівні файлів"
```

File: po/uk.po [hunk 5]
```diff
@@ -12616,11 +12650,15 @@ msgid ""
 "git rebase [-i] [options] [--exec <cmd>] [--onto <newbase> | --keep-base] "
 "[<upstream> [<branch>]]"
 msgstr ""
+"git rebase [-i] [options] [--exec <команда>] [--onto <нова-база> | --keep-"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"git rebase [-i] [опції] [--exec <команда>] [--onto <нова-база> | --keep-"
```

File: po/uk.po [hunk 6]
```diff
@@ -12616,11 +12650,15 @@ msgid ""
...
+"base] [<першоджерельне-сховище> [<гілка>]]"
 
 #: builtin/rebase.c:44
 msgid ""
 "git rebase [-i] [options] [--exec <cmd>] [--onto <newbase>] --root [<branch>]"
 msgstr ""
+"git rebase [-i] [options] [--exec <команда>] [--onto <нова-база>] --root "
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"git rebase [-i] [опції] [--exec <команда>] [--onto <нова-база>] --root "
```

File: po/uk.po [hunk 7]
```diff
@@ -12678,6 +12716,13 @@ msgid ""
 "To abort and get back to the state before \"git rebase\", run \"git rebase --"
 "abort\"."
 msgstr ""
+"Вирішіть всі конфлікти вручну, позначте їх як вирішені за допомогою\n"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Вирішіть усі конфлікти вручну, позначте їх як вирішені за допомогою\n"
```

**Review by arkid15r — COMMENTED** [review]
🏁

File: po/uk.po (hunk 5 above)
Comment (kasya) [reply po/uk.po]: Nice catch!

**Pull request merged**